tribomobile/components/SideBarOption.js

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from 'react';
+import { TouchableHighlight, Text, View, TouchableOpacity } from 'react-native';
+
+const  BarGray = () => {
+  return(
+    <View style={{width: 15}}>
+      <Text> </Text>
+  </View>
+  );
+}
```

Comment:
Identation here could improve, also the Text component is useless??
```suggestion
const BarGray = () => {
  return(
    <View style={{width: 15}}>
      <Text> </Text>
    </View>
  );
}
```

Reply:
Is to make the "15 px" bar width